pkg/logs/input/docker/identifier.go

```diff
@@ -3,8 +3,6 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-2019 Datadog, Inc.
 
-// +build docker
```

Comment:
this should be kept, we just have to create a specific decoder for docker.

Reply:
could you explain why we need to keep this? I understand this is for building, but is it to add more constraint? if we don't put `+build docker`, it won't be included? because the less constraint we have the easier we can test it and linter also can check it(it was ignored by linter).